src/Map/src/Renderer/RendererInterface.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\UX\Map\Renderer;
+
+use Symfony\UX\Map\Map;
+
+/**
+ * @author Hugo Alliaume <hugo@alliau.me>
+ */
+interface RendererInterface extends \Stringable
+{
+    public function renderMap(Map $map, array $attributes): string;
```

Comment:
```suggestion
    public function renderMap(Map $map, array $attributes): string;
```

I'm not sure here.... those runtime attributes only concern the HTML part... not the real Renderer right ?

Reply:
> not the real Renderer

The renderer outputs HTML, I'm not sure to understand

Reply:
Well.... the final method render, no bridge renderer can infere there... so they won't even get the attributes